### PR TITLE
Feature/save client type

### PR DESCRIPTION
**Pull request by Kasturi786**

### CODE REVIEW

File: flaskapp/core/ivr_core.py [hunk 1]
```diff
@@ -373,9 +376,21 @@ def save_data(col_name, value, phone_number, date=None):
     all_data = gs_users_existing.get_all_records()
     if all_data:
         all_data = np.array(all_data)
-        phone_num_index = np.argmax(all_data[:, 0] == phone_number)
-        col_name_index = np.argmax(all_data[0, :] == col_name)
-        gs_users_existing.update_cell(phone_num_index, col_name_index, value)
+        phone_num_index = 1
+        for data in all_data:
+            phone_num_index = phone_num_index + 1
+            print(type(data))
```
Comment (Lina-Lex) [inline flaskapp/core/ivr_core.py]: @Kasturi786 that's great! Quick question, do we need print here? Or this is only for debugging process?

**Review by Lina-Lex — APPROVED** [review]
Good job Murad!!!

**Comment by hazra1991** [comment]
Great work Murad !! .

```python
def save_new_user(username='', phone_number='', tab=''):
    """Function for saving NEW user in google spreadsheet
    and ProstgreSQL database.
```

in the above code function can we make the default arguments as positional arguments .
i feel all the values are required every time we call this function. 

Again thanks for the contribution Excellent work!

**Comment by Lina-Lex** [comment]
Thanks a lot!

**Pull request merged**